pkg/config/config.go

```diff
@@ -224,7 +224,7 @@ type RuntimeHandler struct {
 	AllowedAnnotations []string `toml:"allowed_annotations,omitempty"`
 
 	// DisallowedAnnotations is the slice of experimental annotations that are not allowed for this handler.
-	DisallowedAnnotations []string
+	DisallowedAnnotations []string `toml:"disallowed_annotations,omitempty"`
```

Comment:
@LenkaSeg, so this would be only like the following:

```suggestion
	DisallowedAnnotations []string `toml:"-"`
```

Why? This struct member is not used (or exposed, rather) as a configuration item, it's only used internally within CRI-O, and it was made public so it's easier to reference it (which can be done directly then).

We need to tell TOML library to remove it when it serialises the Go type. And this can be done like in the example above.

Reply:
The most surefire way to ensure that this specific member is never taken into account by the TOML library, would be to make it private, like the `features` member, and then add missing Get() and Set() method for it, so that the value can be set and updated accordingly.

But that is more work.

Reply:
Sure thing!